PythonVisualizations/VisualizationApp.py

```diff
@@ -232,8 +232,7 @@ def animatedOperation():
                     self.cleanUp()
                 command()
             except UserStop as e:
-                while len(self.callStack) > 1:
-                    self.cleanUp(self.callStack[-1])
+                self.cleanUp(self.callStack[0] if self.callStack else None)
```

Comment:
I think this will clean up everything except for the bottom of the stack because of these line in `cleanup`:
```
while len(self.callStack) > minStack: # 1st call unless cleaning all
            top = self.callStack.pop()
            self.cleanUpCallEnviron(top)
            if callEnviron and callEnviron == top: # Stop popping stack if a
                break         # a particular call was being cleaned up
```
This means that drawings are left on the screen after the stop button is pressed. Do we want items still on the screen after pressing the stop button?

Reply:
Yes, the intent of calling `self.cleanUp(self.callStack[0])` is to clean up all but the bottom-most call on the stack.  That leaves the initial call's local variables exposed, showing what the last operation was.  It also leaves the code exposed for review.  Making the initial call's local environment and code disappear would prevent users from studying them after the animation finishes.